docs/source-pytorch/api_references.rst

```diff
@@ -12,7 +12,7 @@ accelerators
 
     Accelerator
     CPUAccelerator
-    GPUAccelerator
+    CUDAAccelerator
     HPUAccelerator
     IPUAccelerator
     TPUAccelerator
```

Comment:
shall be then all other updated as TPU is using XLA?

Reply:
No, we just did this, because we will have multiple GPUs (currently nvidia and mps and in the future likely rocm as well). So we will have a separate accelerator for all of those and `strategy='gpu'` will still work and select among them.

Renaming TPU to XLA would only be required if there was another TPU backend

Reply:
I see just from esthetical point of view it is strange to have AA in sequence lol

Reply:
I agree, but usually people are never expected to use these :) they go with `accelerator='cuda'` or `accelerator='gpu'` which should be fine for your aesthetics :P